catalyst-gateway/bin/src/service/api/documents/put_document/mod.rs

```diff
@@ -195,7 +190,7 @@ async fn store_document_in_db(
     let doc_body = SignedDocBody::new(
         doc.doc_id()?.into(),
         doc.doc_ver()?.into(),
-        doc.doc_type()?.into(),
+        doc.doc_type()?.to_string().parse()?,
```

Comment:
Look at this PR and use `.uuid()` instead https://github.com/input-output-hk/catalyst-libs/pull/439